File: eva-core/eva-core-data-mybatis/src/main/java/io/nerv/core/mybatis/exception/entity/ErrorlogEntity.java
package io.nerv.core.mybatis.exception.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;
import lombok.experimental.Accessors;
import org.apache.ibatis.type.Alias;

/**
 * 异常日志实体类
 */
@Data
@Alias("errorlog")
@TableName("log_error")
@Accessors(chain = true)
public class ErrorlogEntity {
    @TableId(type = IdType.ASSIGN_UUID)
    private String id;
    /**
     * 操作人
     **/
    private String requestTime;
    /**
     * 操作人
     **/
    private String ip;
    /**
     * 操作人
     **/
    private String spendTime;
    /**
     * 操作人
     **/
    private String className;
    /**
     * 操作人
     **/
    private String method;
    /**
     * 操作人
     **/
    private String params;
    /**
     * 操作人
     **/
    private String exDesc;
    /**
     * 操作人
     **/
    private String loginUser;
}

File: eva-core/src/main/kotlin/io/nerv/exception/entity/ErrorlogEntity.kt
package io.nerv.exception.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import com.baomidou.mybatisplus.annotation.TableName
import lombok.Data
import lombok.experimental.Accessors
import org.apache.ibatis.type.Alias

/**
 * 异常日志实体类
 */
@Alias("errorlog")
@TableName("log_error")
class ErrorlogEntity {
    @TableId(type = IdType.ASSIGN_UUID)
    var id: String? = null

    /** 操作人  */
    var requestTime: String? = null

    /** 操作人  */
    var ip: String? = null

    /** 操作人  */
    var spendTime: String? = null

    /** 操作人  */
    var className: String? = null

    /** 操作人  */
    var method: String? = null

    /** 操作人  */
    var params: String? = null

    /** 操作人  */
    var exDesc: String? = null

    /** 操作人  */
    var loginUser: String? = null
}